web/cypress/integration/hfi-calculator-page.spec.ts

```diff
@@ -6,7 +6,7 @@ function interceptDaily(fixturePath: string) {
     cy.intercept('GET', 'api/hfi-calc/daily*', req => {
       const date = new Date(Number(req.query['start_time_stamp']))
       dailies['dailies'].forEach(daily => {
-        daily['date'] = date.toISOString()
+        ;(daily['date'] = date.toISOString()), (daily['last_updated'] = date.toISOString())
```

Comment:
Some invalid syntax here

Reply:
This is being place automatically when I save, I cannot remove it.

Reply:
Try moving the stuff after the comma to a new line, VSCode breaks sometimes.